client/src/config/types.ts

```diff
@@ -112,3 +112,11 @@ export type CastVoteRecordFilesDictionary = Dictionary<CastVoteRecordFile>
 
 export type VoteCounts = Dictionary<Dictionary<number>>
 export type OptionalVoteCounts = Optional<Dictionary<Dictionary<number>>>
+
+export type ISO8601Timestamp = string
+
+export interface PrintedBallot {
+  ballotStyleId: BallotStyle['id']
+  precinctId: Precinct['id']
+  printedAt: ISO8601Timestamp
+}
```

Comment:
what other metadata are you thinking about?

Reply:
I guess I was thinking about `ballotId`, `primaryLocaleCode`, `secondaryLocaleCode`. Maybe `electionHash` too?

Reply:
agree with your first three. `electionHash` not needed as this set of printed ballots is only for the currently configured election, get cleared if you change the election.

I'll update. And add numCopies too.

Reply:
deciding to leave `ballotId` out of it because I'm worried about keeping track of when a particular ID was issued, could lead to voter re-identification. So keeping track of everything except that.